src/modules/formlyBootstrap.js

```diff
@@ -9,10 +9,14 @@ angular.module('formlyBootstrap', ['formly'], function configFormlyVanilla(forml
     {
       name: 'bootstrapHasError',
       templateUrl: 'wrappers/formly-wrappers-bootstrap-has-error.html'
+    },
+    {
+      name: 'bootstrapAddons',
+      templateUrl: 'wrappers/formly-wrappers-bootstrap-addons.html'
     }
   ]);
 
-  var commonWrappers = ['bootstrapLabel', 'bootstrapHasError'];
+  var commonWrappers = ['bootstrapAddons','bootstrapLabel', 'bootstrapHasError'];
```

Comment:
By adding `bootstrapAddons` to the common wrappers, you're adding it to the `radio`, `select`, `input`, and `textarea` types. I don't think this is what we want. We want it to apply to `input` only (correct?)